### ISSUE TITLE
[BUG]: Sibling cards get wrong created date

### ISSUE THREAD

**Issue by brishtibheja** [ankidroid/Anki-Android#16776]
### Checked for duplicates?

- [X] This issue is not a duplicate

### Does it also happen in the desktop version?

- [ ] This bug does not occur in the latest version of Anki

### What are the steps to reproduce this bug?

1. Create a cloze card (card 1)
2. Create another card (card 2)
3. Add another cloze to the first card's note (card 3)
4. View the cards in card browser while display order is set to "by created time"

### Expected behaviour

card 1, card 2, card 3 appears in order.

### Actual behaviour

card 1 and card 3 which are siblings of each other are grouped together.

### Debug info

```text
2.19alpha8
```

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by brishtibheja** [comment]
I realise now the issue might just get solved when card browser starts using the desktop code. Currently, even the creation dates of newly created sibling cards are wrong (which is not in card info).

**Comment by david-allison** [comment]
Listed as potentially fixed by https://github.com/ankidroid/Anki-Android/pull/16620. Once this is merged, please re-test in the next alpha. Thanks!

* https://github.com/ankidroid/Anki-Android/pull/16620
